storefront-nextjs/.env.production

```
@@ -3,3 +3,4 @@ NEXT_PUBLIC_CMS_API_URL=https://api.craftive.io/api
 TENANT_SUBDOMAIN=demo
 TENANT_HOSTNAME=demo.craftive.io
 NEXT_IMAGE_DOMAINS=media.craftive.io
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_IMAGE_DOMAINS key should go before the NEXT_PUBLIC_CMS_API_URL key

(UnorderedKey)